zoom_rooms/indico_zoom_rooms/tasks.py

```diff
@@ -0,0 +1,115 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2024 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from pprint import pformat
+
+import requests
+from celery.schedules import crontab
+
+from indico.core.celery import celery
+from indico.core.db import db
+from indico.core.plugins import current_plugin
+from indico.util.string import strip_control_chars
+
+from indico_zoom_rooms.models import ZoomRoomsAction, ZoomRoomsQueueEntry
+
+
+def _send_request(method: str, zr_id: str, entry_id: str, data: dict | None = None) -> bool:
+    """Send a request to the corresponding HTTP service."""
+    from indico_zoom_rooms.plugin import ZoomRoomsPlugin
+    logger = current_plugin.logger
+
+    path = f'/api/v1/users/{zr_id}/events/{entry_id}'
+    if not (url := ZoomRoomsPlugin.settings.get('service_url')):
+        raise KeyError('service_url is not set!')
```

Comment:
KeyError? Feels a bit weird here...

Reply:
What would you recommend? I chose `KeyError` since it's a miss configuration option, could be seen as a missing key in a dict.